Mention additional requirements to add memfaultd (#271)
memfault/memfault-docs: docs/linux/linux-getting-started-guide.mdx

Include the [`meta-memfault`][source-meta-memfault] layer in your
`bblayers.conf` file.

> Additionally, make sure to make the `/media` directory of your device persists
> upgrades. You can do this by creating a separate partition that's not targeted

Reviewer: ```suggestion
Additionally, make sure to make the `/media` directory and its contents survives
upgrades. You can do this by creating a separate partition that's not targeted
```
It's about the contents of that /media dir, yes?
Should we start with "By default, the SDK writes to various files in /media to store run time configuration, collected data and more. Therefore..."

Is /media "standard"? Do we need to explain what they need to do in case they don't have /media? (i.e. add overrides to memfaultd.conf)

Author: https://github.com/memfault/memfault-docs/pull/271/commits/f8dde37ff4c8dca164ef389c32fe7363380f326a

make
```

> Additionally, make sure to make the `/media` directory of your device persists

Reviewer: Same as above

Author: https://github.com/memfault/memfault-docs/pull/271/commits/f8dde37ff4c8dca164ef389c32fe7363380f326a
